mne/viz/_3d.py

```diff
@@ -1827,12 +1820,15 @@ def plot_source_estimates(stc, subject=None, surface='inflated', hemi='lh',
         transparent=transparent)
     if plot_mpl:
         return _plot_mpl_stc(stc, spacing=spacing, **kwargs)
-    return _plot_stc(
-        stc, overlay_alpha=alpha, brain_alpha=alpha, vector_alpha=alpha,
-        cortex=cortex, foreground=foreground, size=size, scale_factor=None,
-        show_traces=show_traces, src=src, volume_options=volume_options,
-        view_layout=view_layout, add_data_kwargs=add_data_kwargs,
-        brain_kwargs=brain_kwargs, **kwargs)
+    else:
+        with use_3d_backend(backend):
```

Comment:
while you're at it, do you mind improving the [documentation of this function](https://mne.tools/stable/generated/mne.viz.use_3d_backend.html#mne.viz.use_3d_backend)? It should list what are the valid/allowed backends.

Reply:
Also, we discovered that there are 4 backends?

matplotlib, pyvista, mayavi, notebook

Is there some place that lists what needs to be installed for which backend? I'm assuming that not everyone needs "notebook" backend. Similarly "matplotlib" backend name is a bit of a misnomer because it seems to give the impression that you don't need anything other than matplotlib but actually you need nibabel too. I realize this is a bit outside the scope of the PR but things like this stumble poor users :)

Reply:
> Also, we discovered that there are 4 backends?

Actually there are only **three** full fledged 3D backends (`mayavi`, `pyvista` and `notebook`, which is `pyvista` under the hood). This function is an exception since it also supports `matplotlib`. I personally see this as inconsistency but there was a discussion about it not long ago and apparently [this parameter needs some love](https://github.com/mne-tools/mne-python/issues/8394).

This is [what I mean](https://github.com/mne-tools/mne-python/issues/8798#issuecomment-770726019) by:

> The backend parameter in plot_stc does not play nicely with [..] 3d backend. 

Here is what I suggest:

1) More infos are available on [`set_3d_backend`'s doc](https://mne.tools/stable/generated/mne.viz.set_3d_backend.html#mne.viz.set_3d_backend) so I could work on it then link from `use_3d_backend` to there.
2) Work on `plot_stc`'s doc in particular to clarify that **four** values are allowed.

Reply:
+1 on updating `plot_stc`'s doc to indicate four values are allowed!